### ISSUE TITLE
Unidock seems to be not progressing.

### ISSUE THREAD

**Issue by majumderS** [dptech-corp/Uni-Dock#59]
Hi I have run unidock before using the run_dock.py script, it successfully ran. But it has not been running since after that. I have checked the path of unidock : /usr/local/unidock/unidock and checked path of receptor and ligands as well. It just stops at the following screen even for 4-5 ligands.

![image](https://github.com/dptech-corp/Uni-Dock/assets/8430165/490c9424-1874-4bb8-b784-27304cd02948)

/usr/local/unidock/unidock --receptor ./test_files/2eh9.pdbqt --ligand_index def_ligands2.txt --center_x 4.48 --center_y -2.42 --center_z 2.14 --size_x 22 --size_y 22 --size_z 22 --dir ./result2/2eh9 --exhaustiveness 200 --max_step 40 --num_modes 10  --scoring vina --refine_step 3 --seed 5

**Comment by majumderS** [comment]
GPU hows at 100% . Any help would be appreciated.

![image](https://github.com/dptech-corp/Uni-Dock/assets/8430165/3d3293bd-8a05-4462-ba4f-478dd4b2fbdd)

**Comment by ysyecust** [comment]
> Hi I have run unidock before using the run_dock.py script, it successfully ran. But it has not been running since after that. I have checked the path of unidock : /usr/local/unidock/unidock and checked path of receptor and ligands as well. It just stops at the following screen even for 4-5 ligands.
> 
> 
> 
> 
> 
> ![image](https://github.com/dptech-corp/Uni-Dock/assets/8430165/490c9424-1874-4bb8-b784-27304cd02948)
> 
> 
> 
> 
> 
> /usr/local/unidock/unidock --receptor ./test_files/2eh9.pdbqt --ligand_index def_ligands2.txt --center_x 4.48 --center_y -2.42 --center_z 2.14 --size_x 22 --size_y 22 --size_z 22 --dir ./result2/2eh9 --exhaustiveness 200 --max_step 40 --num_modes 10  --scoring vina --refine_step 3 --seed 5
> 
> 

If the number of exhaustiveness is reduced, for example, set to 128, 64. Can the process be completed successfully?

**Comment by majumderS** [comment]
No, it didnt run successfully. It's stuck for one ligand also at Performing docking (random seed: 5) ...

**Comment by ysyecust** [comment]
@majumderS We have updated the code. You can try again. If it still fails, is it convenient to provide the failed source file?
